clang/test/SemaHLSL/BuiltIns/countbits-errors.hlsl

```diff
@@ -0,0 +1,27 @@
+// RUN: %clang_cc1 -finclude-default-header
+// -triple dxil-pc-shadermodel6.6-library %s -fnative-half-type -emit-llvm-only
+// -disable-llvm-passes -verify
+
+
+double test_int_builtin(double p0) {
+  return countbits(p0);
+  // expected-error@-1 {{call to 'countbits' is ambiguous}}
+  // expected-note@hlsl/hlsl_intrinsics.h:* {{candidate function}}
```

Comment:
I don't know if these notes add any value. There are cases where `-verify-ignore-unexpected` is warranted. This might be one of those cases.